Datahub.Core/Components/TopBar/NotificationsList.razor

```diff
@@ -11,15 +11,24 @@
 @using MudBlazor.Utilities
 
 <span>
-    
-    <MudBadge Content=@_unreadNotificationCount Color=@Color.Primary Overlap Visible=@_showNotificationBadge>
```

Comment:
I think this is more useful when it shows the number of unread messages rather than just a dot.